public/kibi_timeline.js

```diff
@@ -184,10 +184,8 @@ define(function (require) {
           const events = [];
 
           if (params.startField) {
-            let startFieldValue;
-            let startRawFieldValue;
-            let endFieldValue;
-            let endRawFieldValue;
+            let startField = { value: undefined, raw: undefined };
+            let endField = { value: undefined, raw: undefined };
```

Comment:
@scampi I would argue that leaving this as it is is more readable and I immediately know what this object should look like when populated
If this is not a problem lets leave it as it is